dnf5.spec

```diff
@@ -172,14 +172,26 @@ It supports RPM packages, modulemd modules, and comps groups & environments.
 
 %files
 %{_bindir}/dnf5
-%{_prefix}/lib/dnf5
-%{_sysconfdir}/dnf/dnf5-aliases.d
-%dir %{_libdir}/dnf5/
-%dir %{_libdir}/dnf5/plugins/
+%doc %{_sysconfdir}/dnf/dnf5-aliases.d/README
+%dir %{_prefix}/lib/dnf5
+%dir %{_prefix}/lib/dnf5/aliases.d
+%config %{_prefix}/lib/dnf5/aliases.d/compatibility.conf
+%dir %{_libdir}/dnf5
+%dir %{_libdir}/dnf5/plugins
 %doc %{_libdir}/dnf5/plugins/README
 %dir %{_datadir}/bash-completion/
 %dir %{_datadir}/bash-completion/completions/
 %{_datadir}/bash-completion/completions/dnf5
+%dir %{_prefix}/lib/sysimage/dnf
+%verify(not md5 size mtime) %ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/groups.toml
+%verify(not md5 size mtime) %ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/modules.toml
+%verify(not md5 size mtime) %ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/nevras.toml
+%verify(not md5 size mtime) %ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/packages.toml
+%verify(not md5 size mtime) %ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/system.toml
+%verify(not md5 size mtime) %ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/transaction_history.sqlite
+%verify(not md5 size mtime) %ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/transaction_history.sqlite-shm
+%verify(not md5 size mtime) %ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/transaction_history.sqlite-wal
+%verify(not md5 size mtime) %ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/userinstalled.toml
```

Comment:
I'd prefer to use a wildcard here: `%ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/*`

Reply:
btw, is `%verify(not md5 size mtime)` needed for config files? (see https://src.fedoraproject.org/rpms/rpm/c/d58b662497ff7adc3be1682143ecb7f101ff1f71?branch=rawhide)